ide/textmate.lexer/external/binaries-list

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+7CDFC30451010E4D6C65AF5A3A4F357563289245 org.jruby.jcodings:jcodings:1.0.41
+655CC3ABA1BC9DBDD653F28937BEC16F3E9C4CEC org.jruby.joni:joni:2.1.11
+041CFB536DA85BCB9DBDF2E465BC952D40704863 org.eclipse.tm4e.core_0.2.0.201809031154.jar
```

Comment:
My personal opinion is that OSGi bundles would better be integrated into the system as OSGi bundles and not as internal wrapped libraries.

Reply:
IMO, the only reason to make them as OSGi bundles would be that other modules would use them. Who are these modules? What is there usecases?

Given I know nothing about compatibility expectations of these projects, do we really want to expose our modules to this unknown compatibility risk? Or is the compatibility of these packages on par with the NetBeans compatibility expectations? Or are we never going to upgrade them?